stock_available/views/product_template_view.xml

```diff
@@ -20,6 +19,15 @@
                             <span class="o_stat_text">Available</span>
                         </div>
                     </button>
+                    <button type="action" name="%(stock.product_open_quants)d"
+                        attrs="{'invisible':[('type', '!=', 'product')]}"
```

Comment:
I tried to use the same visibility as "immediately_usable_qty" field
https://github.com/OCA/stock-logistics-warehouse/pull/192/files/70c8ae8602260ef375d4c765a5d7a02e02eb3e34#diff-33210f15e1b07dbc688105994395937aR14